crates/wasmtime/src/config.rs

```diff
@@ -898,6 +918,33 @@ impl Config {
         self
     }
 
+    /// Allows setting a Cranelift boolean flag or preset. This allows
+    /// fine-tuning of Cranelift settings.
+    ///
+    /// Since Cranelift flags may be unstable, this method should not be considered to be stable
+    /// either; other `Config` functions should be preferred for stability.
+    ///
+    /// # Safety
+    ///
+    /// This is marked as unsafe, because setting the wrong flag might break invariants,
+    /// resulting in execution hazards.
+    ///
+    /// # Errors
+    ///
+    /// This method can fail if the flag's name does not exist.
+    pub unsafe fn cranelift_flag_enable(&mut self, flag: &str) -> Result<&mut Self> {
```

Comment:
I think that this can be done with `cranelift_other_flag(flag, "true")`, right? (not sure if we needed an extra method for this)

Reply:
Unfortunately not; `Configurable::set` and `Configurable::enable` have different functionality relating to presets: the former will error for a preset whereas the latter applies it.

Reply:
Bah ok, well in that case we should probably lean into this naming in all the various places, either `set` or `enable` I think to be consistent about what maps to what